packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -489,10 +489,15 @@ export default class WalletService {
   }
 
   public signWitness = (witness: string, privateKey: string, txHash: string): string => {
+    const witnessArg: CKBComponents.WitnessArgs = {
+      lock: witness,
```

Comment:
Are names for `lock` and `witness` still appropriate?

Reply:
What's your point, I don't quite understand what you mean.